task1-simple-program/get_random_number.c

```diff
@@ -0,0 +1,14 @@
+#include <stdlib.h>
+#include <time.h>
+#include "get_random_number.h"
+
+static unsigned char randomized;
+
+unsigned char get_random_number(unsigned char lower, unsigned char upper)
+{
+	if (!randomized) {
```

Comment:
Each time this function gets called, an initialization check will occur, increasing complexity. Better to avoid such redundancy.